Procurement/ViewModel/SetBuyoutViewModel.cs

```diff
@@ -69,6 +77,30 @@ public void SetBuyoutInfo(ItemTradeInfo info)
             offerInfo.Update(info.CurrentOffer);
             priceInfo.Update(info.Price);
             Notes = info.Notes;
-        }        
+        }
+
+        public ICommand PobDataGenerationCommand => new RelayCommand(GetPobData, CanGetPobData);
+
+        private void GetPobData(object o)
+        {
+            try
+            {
+                Clipboard.SetText(item.PobData);
+
+                //Todo: Inject some sort of display framework - we don't want to spin up 
+                //a message box here...
+                MessageBox.Show("Path of building data copied to the clipboard", "Copied", MessageBoxButton.OK, MessageBoxImage.Information);
```

Comment:
I'd remove the message box, it's going to be more annoying having it pop up than the value of seeing it will provide. We could maybe add something fancy like a border flash for confirmation, but in the meantime I'd go without the message box.

Reply:
Fair comment, I'll work some feedback out inside SetBuyoutViewModel